src/assets/_project/_blocks/layout/footer/_site-footer.scss

```diff
@@ -41,7 +41,7 @@
     }
 
     p {
-        line-height: 20px;
+        line-height: 1.5;
```

Comment:
interesting
https://stackoverflow.com/questions/2040788/whats-the-difference-between-line-height1-5-and-line-height150-in-css